[Service Bus] Enable sending large messages (#23014)
Azure/azure-sdk-for-js: sdk/servicebus/service-bus/CHANGELOG.md

### Bugs Fixed

> - Updating the `sendMessages` API to not batch the messages when it is just a single message. This unblocks the users who are trying to send the large messages(>1MB) using the `sendMessages` API. [#23014](https://github.com/Azure/azure-sdk-for-js/pull/23014)

Reviewer: minor:
```suggestion
- Updating the `sendMessages` API to not batch the messages when it is just a single message. This allows users to send individual large message (>1MB) using the `sendMessages` API. [#23014](https://github.com/Azure/azure-sdk-for-js/pull/23014)
```
